src/index.js

```diff
@@ -28,7 +43,7 @@ const withIs = require('class-is')
  * , as defined in [ipld/cid](https://github.com/multiformats/cid).
  * @class CID
  */
-class CID {
+class CID /*:: <a> */{
```

Comment:
Why is a generic type needed here?

Reply:
The answer is here: https://github.com/multiformats/js-cid/pull/79#discussion_r273327061

Reply:
And will be better documents (tracked here: https://github.com/multiformats/js-cid/issues/84)